website/static/js/fangorn.js

```diff
@@ -1283,15 +1283,19 @@ function _fangornTitleColumn(item, col) {
  */
 function _fangornModifiedColumn(item, col) {
     var tb = this;
+    // Kludge for DropBox date format
+    // TODO: remove kludge when we either move to DropBox v2 API or implememnt
```

Comment:
For normalize yes. [OSF-6461](https://openscience.atlassian.net/browse/OSF-6461) For updating to API v2, I don't believe so.